tests/framework/utils_vsock.py

```diff
@@ -235,12 +235,13 @@ def check_guest_connections(vm, server_port_path, blob_path, blob_hash):
     cmd += "done;"
     cmd += "for w in $workers; do wait $w || exit -1; done"
 
-    ecode, _, _ = vm.ssh.run(cmd)
+    ecode, stdout, stderr = vm.ssh.run(cmd)
 
     echo_server.exit()
     assert echo_server.error is None
 
-    assert ecode == 0, ecode
+    print(stdout.read())
```

Comment:
is this print intended here?

also, probably out of scope here, but there's this annoying inconsistency between `utils.run_cmd` and `ssh.run` where the first returns strings and the second returns these objects that require .read().  :(

Reply:
yep, the `read()` thing I already have it in another patch series. Having a similar interface for those objects would be good too.

Reply:
o, whoop! :D